packages/labs/src/components/tag-input/tagInput.tsx

```diff
@@ -148,13 +154,14 @@ export class TagInput extends AbstractComponent<ITagInputProps, ITagInputState>
 
     public static defaultProps: Partial<ITagInputProps> & object = {
         inputProps: {},
+        inputValue: "",
```

Comment:
:x: remove this default value, it is not correct. the conventional default value for controlled props is `undefined`.

Reply:
Good to know. Removed this line